spinalcordtoolbox/moco.py

```diff
@@ -204,6 +230,11 @@ def moco_wrapper(param):
         im_mask.save(os.path.join(path_tmp, file_mask), mutable=True, verbose=param.verbose)
         # Update field in param (because used later in another function, and param class will be passed)
         param.fname_mask = file_mask
+    if param.fname_ref != '':
+        im_ref = convert(Image(param.fname_ref))
```

Comment:
A note about `convert()`: 

- https://github.com/spinalcordtoolbox/spinalcordtoolbox/issues/3955

Probably out of scope for this PR, but we should refactor this code to no longer use `convert`. I think it's fine to use this function for now just to match the existing conventions.